test/test_ios.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import annotations
+
+import os
+import platform
+import subprocess
+
+import pytest
+
+from . import test_projects, utils
+
+basic_project = test_projects.new_c_project()
+basic_project.files["tests/__init__.py"] = ""
+basic_project.files["tests/__main__.py"] = r"""
+import platform
+import time
+
+# Workaround for CPython#130294
+for i in range(0, 5):
+    time.sleep(1)
+    print("Ensure logger is running...")
+
+print("running tests on " + platform.machine())
+"""
+
+
+def get_xcode_version() -> tuple[int, int]:
+    output = subprocess.run(
+        ["xcodebuild", "-version"],
+        text=True,
+        check=True,
+        stdout=subprocess.PIPE,
+    ).stdout
+    lines = output.splitlines()
+    _, version_str = lines[0].split()
+
+    version_parts = version_str.split(".")
+    return (int(version_parts[0]), int(version_parts[1]))
```

Comment:
This also exists in `test/test_macos_archs.py`, so it can be moved into `test/utils.py` and simply referenced here.